jsapp/js/account/plans/plan.module.scss

```diff
@@ -247,31 +244,86 @@ a.enterpriseLink {
   top: 0;
   border-radius: sizes.$x6 sizes.$x6 0 0;
   height: sizes.$x38;
-  width: 100%;
+  width: variables.$plans-page-card-width;
+  font-weight: 700;
+  font-size: sizes.$x16;
+}
+
+.comparisonButton {
+  width: variables.$plans-page-card-width;
+}
+
+.planButton {
+  height: sizes.$x80;
+  padding-top: sizes.$x20;
 }
 
-@media screen and (max-width: breakpoints.$b1440) {
-  .enterprisePlan {
-    font-size: sizes.$x12;
+.maximizedCards{
+  display: none;
+}
+
+.scrollToTopButton{
+  position: fixed;
+  bottom: sizes.$x20;
+  right: sizes.$x20;
+  border-radius: sizes.$x6;
+  background-color: colors.$kobo-gray-40;
+  width: sizes.$x50;
+  height: sizes.$x40;
+	color: colors.$kobo-white;
+	border: none;
+	cursor: pointer;
+  box-shadow: 0 sizes.$x4 sizes.$x12 0 rgba(0, 0, 0, 0.2);
+}
+
+@media screen and (min-width: breakpoints.$b1000) {
+  .featureContainer {
+    height: 24em;
   }
 
-  .planContainer {
-    font-size: sizes.$x12;
+  .allPlans {
+    column-gap: sizes.$x20;
+    flex-direction: row;
+    width: variables.$plans-page-max-width;
```

Comment:
When there are 2 or fewer plans on the screen, they stick to the left instead of being centered:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/127983ea-c9d9-41bb-a483-74830492ef23)
![image](https://github.com/kobotoolbox/kpi/assets/7819986/9fb3ef1c-23dd-47fb-a2e3-20601b9e7da4)

(That's at `/#/account/plan?type=enterprise`)

It looks like changing the `width` on `.allPlans` to a `max-width` solves that - maybe check that it doesn't have any unintended consequences, though 😅